v2_jobs.go

```diff
@@ -23,6 +25,9 @@ type JobsListRequest struct {
 	Microservice string `json:"microservice,omitempty"`
 	LinkID       string `json:"link_uuid,omitempty"`
 	Name         string `json:"name,omitempty"`
+
+	// WARNING The Detailed query is only available on development branches currently.
```

Comment:
I think that precision is important: `Detailed` is not a query, it's a query parameter.

Here's an alternative. We're confident that this will be included in Archivematica 1.16.

```suggestion
	// Detailed instructs the server to return additional data in the response.
	// This feature is available starting from Archivematica 1.16, which was not
	// released at the time this comment was written.
```